lib/Gitlab/Api/Epics.php

```diff
@@ -0,0 +1,58 @@
+<?php namespace Gitlab\Api;
+
+class Epics extends AbstractApi
+{
+    /**
+     * @param int $group_id
+     * @param array $parameters 
+     *
+     * @return mixed
+     */
+    public function all($group_id, array $parameters = [])
+    {
+        $resolver = $this->createOptionsResolver();
+
+        return $this->get('groups/'.$this->encodePath($group_id).'/-/epics', $resolver->resolve($parameters));
```

Comment:
this is just wrong? there's no such API path like `/-/epics`:
- https://docs.gitlab.com/ee/api/epics.html

I'm sure OP thought web_url paths in gitlab.com web are api paths to be used here.